node/executor/src/lib.rs

```diff
@@ -478,24 +481,24 @@ mod tests {
 		// session change => consensus authorities change => authorities change digest item appears
 		let digest = Header::decode(&mut &block2.0[..]).unwrap().digest;
 		assert_eq!(digest.logs().len(), 0);
-//		assert!(digest.logs()[0].as_consensus().is_some());
+		// assert!(digest.logs()[0].as_consensus().is_some());
```

Comment:
```suggestion
```

Reply:
I assumed maybe it has a meaning and someone left it there on purpose? Can't tell if it is stray or not.